### PR TITLE
(OraklNode) Update docs

### PR DESCRIPTION
**Pull request by nick-bisonai**
# Description

- Documentation updates.
- Removes task `script-test-all` since it is duplicate of task `node`

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Deployment

- [ ] Should publish npm package
- [ ] Should publish Docker image

### CODE REVIEW

File: node/README.md [hunk 1]
```diff
@@ -105,13 +109,19 @@ ETH_PROVIDER_URL=<Provider URL>
...
-# Delegator URL, tx fee is directly payed from reporter if not provided
+# Delegator URL, tx fee is directly paid from reporter if not provided
 DELEGATOR_URL=<Your Delegator URL>
 
+# Signer PK, generates a signature based on this value
+SIGNER_PK=<Your Signer PK>
+
+# Encrypt Password, this is referenced to store encrypted wallet pk into table. defaults to 'anything'
+ENCRYPT_PASSWORD=<Your Encrypt Password>
+
```
[the remark this answers is not in the document]
Comment (nick-bisonai) [reply node/README.md:120]: What kind of more detailed explanation do you expect? what is lacking in current description?

File: node/README.md [hunk 2]
```diff
@@ -26,13 +30,12 @@ Off-chain aggregator performs the following steps to regularly submit data into
 2. Send and receive data with other nodes, aggregate all received data, and save aggregated data into the database.
 3. Submit aggregated data into the chain.
 
-![Overview](./Node.drawio.svg)
 
-- Set of `Admin`, `fetcher`, `aggregator`, and `reporter` runs in a single Orakl Node
+<figure><img src="./Node.drawio.svg" alt=""><figcaption><p>Set of `Admin`, `fetcher`, `aggregator`, and `reporter` runs in a single Orakl Node</p></figcaption></figure>
```
[the remark this answers is not in the document]
Comment (nick-bisonai) [reply node/README.md:34]: I need captions so I can't take your suggestion

File: node/README.md [hunk 3]
```diff
@@ -128,38 +138,71 @@ BOOT_API_PORT=<Your Boot API Port>
...
 ```sh
 migrate -database "{$DATABASE_URL}" -path ./migrations up
 ```
 
-### Test Run Single Local Node
+This process will generate required tables and constraints to run the service.
+
+### Optional Setups
+
+Following are optional setups which helps the application run more smoothly."
```
Comment (martinkersner) [inline node/README.md:160]:
```suggestion
Following are optional setups which helps the application run more smoothly.
```

**Review by martinkersner — APPROVED** [review]
LGTM!

**Pull request merged**